.github/workflows/_runner_ondemand_slurm.yaml

```diff
@@ -98,9 +99,26 @@ jobs:
           echo "SLURM_JOB_ID=$(cat ${SLURM_JOB_ID_FILE})" >> $GITHUB_OUTPUT
 
       - name: Wait for SLURM job to complete
+        id: job-completion
         shell: bash -x -e {0}
         run: |
           while ssh ${{ secrets.CLUSTER_LOGIN_USER }}@${{ vars.HOSTNAME_SLURM_LOGIN }} squeue -j${{ steps.submit.outputs.SLURM_JOB_ID }} > /dev/null 2>&1; do echo "wait"; sleep 15; done
+          SLURM_JOB_COMPLETION_STATUS_FILE=$(mktemp)
+          ssh ${{ secrets.CLUSTER_LOGIN_USER }}@${{ vars.HOSTNAME_SLURM_LOGIN }} >${SLURM_JOB_COMPLETION_STATUS_FILE} \
+            sacct -XP -j ${{ steps.submit.outputs.SLURM_JOB_ID }}
+
+          echo "JOB COMPLETION STATUS"
+          cat ${SLURM_JOB_COMPLETION_STATUS_FILE}
+
+          SLURM_JOB_COMPLETION_STATE_FILE=$(mktemp)
+          ssh ${{ secrets.CLUSTER_LOGIN_USER }}@${{ vars.HOSTNAME_SLURM_LOGIN }} >${SLURM_JOB_COMPLETION_STATE_FILE} \
```

Comment:
Why repeat this? Can we `cut -d| -f<N>` from the previous result?

Reply:
Done